torch/csrc/jit/graph_executor.cpp

```diff
@@ -72,6 +73,17 @@ struct ExecutionPlanAutogradFunction : public autograd::Function {
 };
 
 
+// helper to run interpreter on variables until we switch
+// everything to IValue
+inline variable_tensor_list runOneStage(const Code & code, variable_tensor_list inputs) {
```

Comment:
I guess now that we don't have stages we could just call it `runWithVariables`

Reply:
The interpreter can still compute stages, and it adds complexity to the it as well. We should add a task to remove this functionality and then rename this function.